commands/create/cluster/command.go

```diff
@@ -250,9 +252,16 @@ func printResult(cmd *cobra.Command, positionalArgs []string) {
 		case errors.IsNotEnoughWorkerNodesError(err):
 			headline = "Not enough worker nodes specified"
 			subtext = fmt.Sprintf("If you specify workers in your definition file, you'll have to specify at least %d worker nodes for a useful cluster.", limits.MinimumNumWorkers)
+		case errors.IsYAMLNotParseableError(err):
```

Comment:
FYI, by convention the matchers do not have the `Error` suffix. Check our `fmt` and all operator code.

Reply:
Some detail to pick up every day! :)